IM_calculation/IM/im_calculation.py

```diff
@@ -282,9 +282,10 @@ def calc_FAS(
         )
     else:
         # compute EAS, the euclidean distance of FAS 000 and 090
-        values_to_store[
-            Components.ceas.str_value
-        ] = intensity_measures.get_euclidean_dist(value[:, 0], value[:, 1])
+        if Components.ceas in comps_to_store:
+            values_to_store[
+                Components.ceas.str_value
+            ] = intensity_measures.get_euclidean_dist(value[:, 0], value[:, 1])
```

Comment:
Does this only calculate EAS for FAS? It looks like it calculated the euclidian distance for all IMs? But I'm happy to be proven wrong.

Reply:
Not sure of your question. This is a part of calc_FAS function, and computes Euclidean distance for FAS.

Reply:
Oh it does too. I see how that works now. I presumed it was in the part near where geom was calculated.